crates/starknet/src/constants.rs

```diff
@@ -27,6 +27,13 @@ pub(crate) const UDC_CONTRACT_CLASS_HASH: &str =
 pub(crate) const UDC_CONTRACT_ADDRESS: &str =
     "0x41A78E741E5AF2FEC34B695679BC6891742439F7AFB8484ECD7766661AD02BF";
 
+pub const DECLARE_V1_TRANSACTION_HASH: &str =
```

Comment:
Why would you put these constants here?

Reply:
I overreacted.

Moved to to `get_transaction_by_hash.rs` test file in 8da587c130635f41ab7f364a1ce2b76b51a7c2f6